### ISSUE TITLE
Can we use this addon to make a Websocket client?

### ISSUE THREAD

**Issue by Meach** [bakercp/ofxHTTP#53]
Hello,

I want to use websockets and make a client that would connect to my websocket server made with nodejs.
I see the readme note about using ofxJSONRPC to use websockets. Thing is: I really don't want to use JSON RPC, only bare websockets.
I had a look at the code but I do not understand how to make a simple websocket client.

Any hints or example will be great. 

Thanks!

**Comment by bakercp** [comment]
So to be clear I understand, you want to connect from an OF app to web
server with web sockets (as a client)?

ofxJSONRPC is an a way of making an oF server.

**Comment by Meach** [comment]
Yes, in the past I used ofxJSONRPC to make this web server and everything was bundled within OF.

Now I want to separate the web part and use nodejs to take care of the webserver and websocket server. OF will only have a websocket client to communicate with it. I thought ofxHTTP is a good starting point for this but looking at the source makes me very confused. I cannot understand how I could make a websocket client app.

Otherwise I found this [websocket client example with Poco](https://gist.github.com/tuxmartin/1218851b7e025f68ecc50f949c9dd332) and was thinking to maybe use it as a starting point if I cannot make a websocket client with your addons.

**Comment by stephanschulz** [comment]
Have you ever found a solution to making a Websocket client with ofxHTTP?
I tired ofxLibwebsockets but it collides with my other use of ofxHTTP and ofxSSLManager.

**Comment by Meach** [comment]
I ended up using [ofxSocketIO](https://github.com/soixantecircuits/ofxSocketIO) for this purpose. I don't know if it will create some trouble with ofxHTTP or ofxSSLManager as I don't use them anymore.

**Comment by stephanschulz** [comment]
I ended up using a simple websocket with poco and ofxHTTP.
https://github.com/stephanschulz/OBS_OF_websocket
